packages/mds-stream-processor/labelers/telemetry-labeler.ts

```diff
@@ -0,0 +1,47 @@
+import { Timestamp, Nullable, Telemetry, NullableProperties } from '@mds-core/mds-types'
+import { MessageLabeler } from './types'
+
+export interface TelemetryLabel {
+  telemetry_timestamp: Timestamp
+  telemetry_lat: number
+  telemetry_lng: number
+  telemetry_altitude: Nullable<number>
+  telemetry_heading: Nullable<number>
+  telemetry_speed: Nullable<number>
+  telemetry_accuracy: Nullable<number>
+  telemetry_charge: Nullable<number>
+}
+
+export const TelemetryLabeler: () => MessageLabeler<{ telemetry: Telemetry }, TelemetryLabel> = () => async ({
+  telemetry
+}) => {
+  const { timestamp, gps } = telemetry
+  return {
+    telemetry_timestamp: timestamp,
```

Comment:
I think for cases of vehicle events telemetry_timestamp will be identical to event_timestamp. To be concise with each record  I would recommend getting rid of telemetry_timestamp from the OptionalTelemetryLabeler all together and explicitly assign it as a field in the vehicle-telemetry-processor

Reply:
I don't think they are _guarnateed_ to be identical. If they were, we would not be storing _telemetry_timestamp_ on an event in PG because we could join using event.timestamp. Events, should have both, IMO.

Reply:
They are not guaranteed to be the same, that is correct. Hence why it's necessary to keep that annotation.